lib/rspec/rails/fixture_support.rb

```diff
@@ -29,6 +29,27 @@ module FixtureSupport
           end
           self.use_instantiated_fixtures  = RSpec.configuration.use_instantiated_fixtures
           fixtures RSpec.configuration.global_fixtures if RSpec.configuration.global_fixtures
+
+          def self.fixtures(*args)
+            orig_methods = private_instance_methods
+            super.tap {
+              new_methods = private_instance_methods - orig_methods
+              new_methods.each do |method_name|
+                proxy_method_warning_if_called_in_before_context_scope(method_name)
+              end
+            }
+          end
+
+          def self.proxy_method_warning_if_called_in_before_context_scope(method_name)
+            orig_implementation = instance_method(method_name)
+            define_method(method_name) do |*args, &blk|
+              if self.inspect.include?("before(:context)")
```

Comment:
I could not think of a better one after staring at the source... maybe @myronmarston has thoughts?